src/components/Choice/Choice.scss

```diff
@@ -10,6 +10,7 @@ $control-vertical-adjustment: rem(2px);
   display: inline-flex;
   justify-content: flex-start;
   padding: spacing(extra-tight) 0;
+  cursor: pointer;
```

Comment:
I like this! However, since a choice can be disabled I think it could lead to confusion. It might be best to set it back to default when disabled.